pysph/cpy/translator.py

```diff
@@ -672,14 +673,15 @@ def ocl_detect_type(name, value):
     if isinstance(value, KnownType):
         return value.type
     elif name.startswith(('s_', 'd_')) and name not in ['s_idx', 'd_idx']:
-        return '__global double*'
+        return 'GLOBAL_MEM double*'
```

Comment:
This will require including cluda_preamble in the `acceleration_eval_opencl_helper.py`.  The other opencl tests will fail with this change.